### PR TITLE
Polishing sentiment role labeling using paid Grammeraly

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

**Comment by wangkuiyi** [comment]
Please wait till I correct the identation.

File: label_semantic_roles/README.en.md [hunk 1]
```diff
@@ -52,21 +52,21 @@ Fig 2. BIO represention
...
 
-In this tutorial, our SRL system is built as an end-to-end system via neural network. We take only text sequences, without using any syntactic parsing results or complex hand-designed features. We give public dataset [CoNLL-2004 and CoNLL-2005 Shared Tasks](http://www.cs.upc.edu/~srlconll/) as an example to illustrate: given a sentence and it's predicates, identify the corresponding arguments and their semantic roles by sequence tagging method.
+In this tutorial, our SRL system is built as an end-to-end system via a neural network. We take only text sequences, without using any syntactic parsing results or complex hand-designed features. We give public dataset [CoNLL-2004 and CoNLL-2005 Shared Tasks](http://www.cs.upc.edu/~srlconll/) as an example to illustrate: given a sentence and it's predicated, identify the corresponding arguments and their semantic roles by sequence tagging method.
```
Comment (helinwang) [inline label_semantic_roles/README.en.md]: predicates in it

File: label_semantic_roles/README.en.md [hunk 2]
```diff
@@ -92,10 +92,10 @@ Fig 4. Bidirectional LSTMs
 
 线性变换-> linear transformation
 输入层到隐层-> input-to-hidden
-正向处理输出序列->process sequence in forward direction
-反向处理上一层序列-> process sequence from previous layer in backward direction
+正向处理输出序列->process sequence in forwarding direction
```
Comment (helinwang) [inline label_semantic_roles/README.en.md]: typically people use `forward direction ` in the literature. I have not see `forwarding direction` before

File: label_semantic_roles/README.en.md (hunk 1 above)
Comment (wangkuiyi) [reply label_semantic_roles/README.en.md]: done

File: label_semantic_roles/README.en.md (hunk 2 above)
Comment (wangkuiyi) [reply label_semantic_roles/README.en.md]: done

File: label_semantic_roles/README.en.md [hunk 3]
```diff
@@ -148,9 +148,9 @@ We can try above method. Here, we propose some modifications by introducing two
 After modification, the model is as follows:
 
 1. Construct inputs
- - Input 1: word sequence. Input 2: predicate. Input 3: predicate context, extract $n$ words before and after predicate. Input 4: region mark sequence, element value will be 1 if word locates in the predicate context region, 0 otherwise.
+ - Input 1: word sequence. Input 2: predicate. Input 3: predicate context, extract $n$ words before and after predicate. Input 4: region mark sequence, element value will be one if word locates in the predicate context region, 0 otherwise.
```
Comment (helinwang) [inline label_semantic_roles/README.en.md]: there is a `0` latter in the sentence, so please either `one, zero`, or `1, 0`

File: label_semantic_roles/README.en.md [hunk 4]
```diff
@@ -148,9 +148,9 @@ We can try above method. Here, we propose some modifications by introducing two
...
  - expand input 2~3 as sequences with the same length with input 1
-2. Convert input 1~4 to vector sequences via lookup table; input 1 and 3 shares the same lookup table, input 2 and 4 have separate lookup tables
+2. Convert input 1~4 to vector sequences via lookup table; input 1 and three shares the same lookup table, input 2 and 4 have separate lookup tables
```
Comment (helinwang) [inline label_semantic_roles/README.en.md]: I think 3 is better, in this context steps are used as arabic numbers.

File: label_semantic_roles/README.en.md (hunk 3 above)
Comment (wangkuiyi) [reply label_semantic_roles/README.en.md]: done

File: label_semantic_roles/README.en.md (hunk 4 above)
Comment (wangkuiyi) [reply label_semantic_roles/README.en.md]: done

**Review by helinwang — APPROVED** [review]
LGTM

**Pull request merged**